kong/api/routes/certificates.lua

```diff
@@ -12,6 +12,37 @@ return {
         self.params.snis = nil
       end
 
+      -- dont add the certificate or any snis if we have an SNI conflict
+      -- its fairly inefficient that we have to loop twice over the datastore
+      -- but no support for OR queries means we gotsta!
+      local seen = {}
```

Comment:
I believe this `seen` table does not have to belong to this scope, but can be scoped under the `if snis then` block?